Source/WebCore/platform/graphics/MediaPlayer.cpp

```diff
@@ -639,6 +639,10 @@ void MediaPlayer::loadWithNextMediaEngine(const MediaPlayerFactory* current)
                 m_private->startVideoFrameMetadataGathering();
             if (m_processIdentity)
                 m_private->setResourceOwner(m_processIdentity);
+#if ENABLE(LEGACY_ENCRYPTED_MEDIA) && ENABLE(ENCRYPTED_MEDIA)
+            if (m_shouldContinueAfterKeyNeeded)
```

Comment:
the test is not needed, I think it's nicer to send the value all the time to ensure up to date data.

This will send it to the RemoteMediaPlayerProxy which will send it to its MediaPlayer and the code as-is is sufficient